### DISCUSSION TITLE
Feature request: Add stats granularity control (days, weeks, months)

### DISCUSSION THREAD

**Discussion by anmipo** [plausible/analytics#1210] [Anything Else]
Currently, the "mid-term" time window options are limited to:

- Last 30 days (by day)
- Last 6 months (by month)

The gap between them is quite wide... 

Normally, I check my stats over the last 90 days, by week. I can quickly select the nearest option (6 month) but it shows monthly values — which is too coarse. So I have to fallback to the custom range — but it shows daily values. 
So every time I have to 1) click many times to pick the 90-day window, and 2) guestimate weekly values by scientifically squinting my eyes :raised_eyebrow:

It would be great to have a quick selector for daily/weekly/monthly stats...

**Comment by Vigasaurus** [comment]
This probably ought to be a discussion given it is a feature request and not a bug, but either way, I am working on this as part of my major changes to the main graph (including new tooltips, comparisons, metric selection, and this option to choose granularity).

**Comment by adrianmsmith** [comment]
I wanted to ask about the status of this "group by week" feature. Looking at the live demo at https://plausible.io/plausible.io it doesn't seem that group by week is available? For sites targeting businesses there is a lot more traffic Mon-Fri and much less at the weekend, so group by day gives a very bumpy graph, and the numbers on group by month really depend on how many weekend days there are in the month. Group by week solves these problems. This is a feature I used all the time on Google Analytics and https://pirsch.pirsch.io/ seems to support it as well. Thanks in advance!

**Reply by metmarkosaric** [reply]
hi @adrianmsmith! we support this already. there's an option to change the time interval on the right-hand side of the top chart. like this:

![Screenshot from 2023-04-24 18-55-57](https://user-images.githubusercontent.com/34340819/234064856-3e8c1bca-76c0-4ff0-a91e-2232f5b17aaf.png)

**Reply by adrianmsmith** [reply]
Ah, perfect! I looked around for ages but didn’t spot it. Thanks! (And thanks for your very fast reply!)
